CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Changelog
 
 
+### NEXT
+
+* Fix `SeqManager`: Properly consider previous cycle dropped inputs ([PR #1032](https://github.com/versatica/mediasoup/pull/1032)).
+
```

Comment:
Second attempt to put 2 blank lines between entries in this file :)

```suggestion
* Fix `SeqManager`: Properly consider previous cycle dropped inputs ([PR #1032](https://github.com/versatica/mediasoup/pull/1032)).


```

Reply:
OMG